FinniversKit/Sources/Components/BrazePromotionView/BrazePromotionView.swift

```diff
@@ -151,29 +151,33 @@ public class BrazePromotionView: UIView {
     }
 
     private func loadImage() {
-        guard let imageUrl = viewModel.image else {
-            remoteImageView.removeFromSuperview()
-            NSLayoutConstraint.deactivate(stackViewConstraintsImage)
-            NSLayoutConstraint.activate(stackViewConstraintsNoImage)
+        if let imageUrl = viewModel.image {
+            closeButton.setImage(UIImage(named: .cross), for: .normal)
+            closeButton.backgroundColor = UIColor.black.withAlphaComponent(0.6)
```

Comment:
Is dark mode tested with this?

Reply:
Yes, the color is only black when there is an image there, so that's why the color is the same for light and dark mode here. Good observation 😊